### PR TITLE
Added Wordpress version 6.15.0.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16435)

### CODE REVIEW

File: manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Moniker: wordpress
+Tags:
+- admin
+- electron
+- wordpress
+- blog
+- editor
+PackageUrl: https://apps.wordpress.com/desktop/
+License: GPLv2
+LicenseUrl: https://raw.githubusercontent.com/Automattic/wp-calypso/v6.11.0/LICENSE.md
```
Comment (ItzLevvie2) [inline manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml:16]:
```suggestion
LicenseUrl: https://raw.githubusercontent.com/Automattic/wp-calypso/v6.15.0/LICENSE.md
```

File: manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Automattic.Wordpress
+Publisher: Automattic
+PackageName: Wordpress
```
Comment (ItzLevvie2) [inline manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml:4]:
```suggestion
PackageName: WordPress.com
```
image

File: manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Automattic.Wordpress
+Publisher: Automattic
```
Comment (ItzLevvie2) [inline manifests/a/Automattic/Wordpress/6.15.0/Automattic.Wordpress.yaml:3]:
```suggestion
Publisher: Automattic Inc.
```

**Review by ghost — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Comment by jedieaston** [comment]
I need to do some serious bug fixing on my script. Thanks for catching this.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**